app/src/main/java/cm/aptoide/pt/analytics/view/AnalyticsActivity.java

```diff
@@ -41,18 +38,18 @@ public boolean is_resumed() {
           .getLanguage());
     }
 
-    Analytics.Dimensions.setGmsPresent(AdNetworkUtils.isGooglePlayServicesAvailable(this));
+    ((AptoideApplication) getApplicationContext()).getFirstLaunchAnalytics()
+        .setGmsPresent(AdNetworkUtils.isGooglePlayServicesAvailable(this));
```

Comment:
Can we inject the FirstLaunchAnalytics dependency?

Reply:
Fixed. The problem was that it was assuming the class was ActivityResultNavigator with "this", as a result it wasn't injecting the dependecies because the ActivityResultNavigator was already injected.